StreamDeckPlugin/Services/SendEventHandler.cs

```diff
@@ -0,0 +1,125 @@
+﻿using ArkhamOverlay.TcpUtils;
+using ArkhamOverlay.TcpUtils.Requests;
+using ArkhamOverlay.TcpUtils.Responses;
+using Newtonsoft.Json;
+using StreamDeckPlugin.Events;
+using StreamDeckPlugin.Utils;
+
+namespace StreamDeckPlugin.Services {
+    public interface ISendEventHandler {
+    }
+
+    public class SendEventHandler : ISendEventHandler {
+        private readonly IEventBus _eventBus;
+        private readonly IDynamicActionInfoStore _dynamicActionInfoStore;
+        private readonly IImageService _imageService;
+
+        public SendEventHandler(IEventBus eventBus, IDynamicActionInfoStore dynamicActionInfoStore, IImageService imageService) {
+            _eventBus = eventBus;
+            _dynamicActionInfoStore = dynamicActionInfoStore;
+            _imageService = imageService;
+
+            eventBus.OnRegisterForUpdates(RegisterForUpdates);
+            eventBus.OnClearAllCards(ClearAllCards);
+            eventBus.OnGetButtonInfo(GetCardInfo);
+            eventBus.OnDynamicButtonClicked(DynamicButtonClicked);
+            eventBus.OnGetInvestigatorImage(GetInvestigatorImage);
+            eventBus.OnShowDeckList(ShowDeckList);
+            eventBus.OnTakeSnapshot(TakeSnapshot);
+            eventBus.OnGetStatValue(GetStatValue);
+            eventBus.OnChangeStatValue(ChangeStatValue);
+            eventBus.OnGetButtonImage(GetButtonImage);
+        }
+
+        private string SendRequest(Request request) {
+            return SendSocketService.SendRequest(request, TcpInfo.ArkhamOverlayPort);
+        }
+
+        private T SendRequest<T>(Request request) where T : Response {
```

Comment:
Doesn't seem like we're using the return on this right now, so maybe we can combine with the above SendRequest method

Reply:
This allows the calls that don't care about the response to execute without having to put a response type in.

Example: SendRequest(request);